tools/dependencies/known-dependencies.txt

```diff
@@ -345,3 +345,14 @@ protobuf-java-util-3.17.2.jar
 zjsonpatch-0.3.0.jar
 zookeeper-3.8.0.jar
 zookeeper-jute-3.8.0.jar
+aliyun-java-sdk-core-4.5.10.jar
+aliyun-java-sdk-kms-2.11.0.jar
+aliyun-java-sdk-ram-3.1.0.jar
```

Comment:
You forgot announce `aliyun-java-sdk-ram` in LICENSE?

Reply:
Fixed in the latest commit. Thanks for pointing this out.